_episodes/14-supplemental-vscode.md

```diff
@@ -0,0 +1,101 @@
+---
+title: "Supplemental: Using Git from VS-Code"
```

Comment:
Can the title be more explanatory? Perhaps: "Using Git with an Integrated Development Environment (Visual Studio Code)". Best not to assume that participants are familiar with VS-Code or the difference between IDEs and text editors.

Reply:
I would argue that VS-code is more text editor than IDE and can be made into an IDE with extensions. We are not covering this and I think that associating VS-Code with the IDE's confuses things, especially for beginners who are unfamiliar with  the concept as you say. Its just a tool for writing code and interacting with git. Lets treat it as such.

Reply:
actually you are right